images/dockerregistry/config.yml

```diff
@@ -35,3 +35,7 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
```

Comment:
Upstream has a configuration section called [reporting](https://docs.docker.com/registry/configuration/#reporting). This could be probably nested there - similarly to the `auth`.